addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/command/UdpCubeCommand.java

```diff
@@ -152,15 +152,6 @@ private void receiveUdpCommandResponse() {
         } catch (IOException e) {
             logger.debug("IO error during MAX! Cube response: {}", e.getMessage());
             commandRunning = false;
-        } finally {
```

Comment:
I changed it to the Java 7 
``` try (closable = SomethingCloseable) { block; } ``` syntax. This takes care of checking for nullability as well as exception handling.